chain/actors/actor_miner.go

```diff
@@ -734,9 +748,68 @@ func (sma StorageMinerActor) GetSectorSize(act *types.Actor, vmctx types.VMConte
 	return types.NewInt(mi.SectorSize).Bytes(), nil
 }
 
-type PaymentVerifyParams struct {
-	Extra []byte
-	Proof []byte
+func isLate(height uint64, self *StorageMinerActorState) bool {
+	return self.ProvingPeriodEnd > 0 && height >= self.ProvingPeriodEnd // TODO: review: maybe > ?
```

Comment:
Yeah, i'd go just >

Reply:
we should set the genesis miners proving period end to be several days after genesis

Reply:
Thinking about this more - we are calling this after all messages in that tipset were applied, `>=` still allows the miner to submit post in the last block of proving period.

The difference between `>=`, and `>` is that `>=` slashes the miner immedietly after proving period with no post,  `>` will wait one more proving period before slashing

Reply:
(I think I prefer `>=`)

Reply:
Thats a very fair point that I agree with, fine with me to keep as is